tests/sys/crypto/tests-crypto-chacha20poly1305.c

```diff
@@ -31,7 +31,7 @@ static const uint8_t key_1[32] = {
     0x90, 0x91, 0x92, 0x93, 0x94, 0x95, 0x96, 0x97, 0x98, 0x99, 0x9a, 0x9b, 0x9c, 0x9d, 0x9e, 0x9f
 };
 
-static const uint8_t msg_1[] = {
+static const uint8_t msg_1[114] = {
```

Comment:
Why was it necessary to add those numbers explicitely? I'd rather leave them out.